build.sbt

```diff
@@ -76,6 +76,16 @@ lazy val noop = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   )
   .nativeSettings(commonNativeSettings)
 
+lazy val ce3 = crossProject(JSPlatform, JVMPlatform)
+  .settings(commonSettings)
+  .dependsOn(core)
+  .settings(
+    name := "log4cats-ce3",
```

Comment:
I would call this `log4cats-cats-effect`.  Anything beyond `3` will be a breaking change, and I'd avoid colloquial abbrevations in a jar name.